files/build_templates/docker_image_ctl.j2

```diff
@@ -238,7 +258,15 @@ start() {
             redis_dir=`echo $redis_dir_list | cut -d " " -f $id`
             REDIS_MNT=" -v $redis_dir:$redis_dir:rw "
         fi
-
+        {%- if '--log-driver=json-file' in docker_image_run_opt %}
+            LOG_OPTS="--log-opt max-size=2M --log-opt max-file=5"
+        {%- else %}
+            # Containers in the docker network will use the syslog-driver as log-driver. This is mainly applicable for Multi NPU system
+            # In this case the syslog service on the host is listening on the docker0 address
+            syslog_address=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1)
```

Comment:
```suggestion
            syslog_address=$(docker network inspect bridge --format='{{(index .IPAM.Config 0).Gateway}}')
```

Reply:
This IP address is actually available in build time. Check `files/docker/docker.service.conf`

---
In reply to: [439565102](https://github.com/Azure/sonic-buildimage/pull/4738#discussion_r439565102) [](ancestors = 439565102)

Reply:
This change has been removed